Philips.CodeAnalysis.MaintainabilityAnalyzers/Naming/NamespaceMatchFilePathAnalyzer.cs

```diff
@@ -72,7 +73,7 @@ private bool IsNamespacePartOfPath(string ns, string path)
 			return false;
 		}
 
```

Comment:
Did you make these static because of sonar? In general, I thought we moved away from this pattern because it restricts unit testing (ie ability to mock).

Reply:
Yes, this was coming from SanorCloud. I didn't see any impact on tests for this change

Reply:
Yes it's more of a general thing. Many years ago there was a CA rule for exactly this, and we had thousands of methods marked static as a result. But as we added mocks for unit tests, we hit a wall. As a result, we disabled this rule and removed "static" wherever possible. I.e., the conclusions was "static is bad". In fact, I thought we had a rule in this library called "Avoid statics" that I'd've thought the dogfooding would catch...

Reply:
We indeed have a rule about Static classes, which is why the `Helper.For...` syntax was born.